UE4SS/src/GUI/LiveView.cpp

```diff
@@ -2950,31 +3081,123 @@ namespace RC::GUI
+                        {
+                            std::erase_if(property_name, isspace);
+                            Filter::HasProperty::list_properties.emplace_back(to_wstring(property_name));
+                        }
+                    }
                 }
 
                 // Row 7
                 ImGui::TableNextRow();
                 ImGui::TableNextColumn();
                 ImGui::Text("Has property of type");
                 ImGui::TableNextColumn();
-                if (ImGui::InputText("##HasPropertyType", &Filter::HasPropertyType::s_internal_value))
+                ImGui::SetNextItemWidth(ImGui::GetContentRegionAvail().x);
+                if (ImGui::InputText("##HasPropertyType", &Filter::HasPropertyType::s_internal_property_types))
                 {
-                    Filter::HasPropertyType::s_value = FName(to_wstring(Filter::HasPropertyType::s_internal_value), FNAME_Add);
+                    Filter::HasPropertyType::list_property_types.clear();
+                    if (!Filter::HasPropertyType::s_internal_property_types.empty())
+                    {
+                        std::istringstream ss(Filter::HasPropertyType::s_internal_property_types);
+                        std::string property_type_name;
+                        while (std::getline(ss, property_type_name, ','))
+                        {
+                            std::erase_if(property_type_name, isspace);
+                            if (!property_type_name.empty())
+                            {
+                                Filter::HasPropertyType::list_property_types.emplace_back(FName(to_wstring(property_type_name), FNAME_Add));
+                            }
+                        }
+                    }
+                }
+                
+                ImGui::TableNextRow();
+                ImGui::TableNextColumn();
+                if (ImGui::Button(ICON_FA_SEARCH " Refresh search"))
+                {
+                    if (listeners_allowed)
+                    {
+                        std::string search_buffer{m_search_by_name_buffer};
+                        if (search_buffer.empty())
+                        {
+                            Output::send(STR("Search all chunks\n"));
+                            s_name_to_search_by.clear();
+                            m_object_iterator = &LiveView::guobjectarray_iterator;
+                            m_is_searching_by_name = false;
+                        }
+                        else
+                        {
+                            Output::send(STR("Search for: {}\n"), search_buffer.empty() ? STR("") : to_wstring(search_buffer));
+                            s_name_to_search_by = search_buffer;
+                            m_object_iterator = &LiveView::guobjectarray_by_name_iterator;
+                            m_is_searching_by_name = true;
+                            search_by_name();
+                        }
+                    }
```

Comment:
This is code duplication.
Can we make this into a member function that we can use here and at the original location ?